tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/internal/types/TBfloat16Mapper.java

```diff
@@ -0,0 +1,43 @@
+package org.tensorflow.internal.types;
+
+import org.tensorflow.TensorMapper;
+import org.tensorflow.DataType;
+import org.tensorflow.RawTensor;
+import org.tensorflow.internal.buffer.TensorBuffers;
+import org.tensorflow.ndarray.buffer.FloatDataBuffer;
+import org.tensorflow.ndarray.buffer.layout.DataLayouts;
+import org.tensorflow.ndarray.impl.dense.FloatDenseNdArray;
+import org.tensorflow.types.TBfloat16;
+
+/**
+ * Maps memory of {@link org.tensorflow.proto.framework.DataType#DT_BFLOAT16} tensors
+ * to a n-dimensional data space.
+ */
+public class TBfloat16Mapper extends TensorMapper<TBfloat16> {
+
+  @Override
+  public TBfloat16 mapDense(RawTensor tensor) {
```

Comment:
Did you mean to change the access level of `mapDense` here? It's protected in the super class and the other implementations.

Reply:
ah true thanks, I'll keep it protected in subclasses as well, good catch!